core/definitions/src/testing/mocks/platform.ts

```diff
@@ -40,6 +40,7 @@ export function mockPlatformFactory<P extends PlatformName>(
 export class MockPlatform<P extends PlatformName> implements Platform<P> {
   // @ts-ignore
   readonly platform: P;
+  readonly nativeDecimals = 18n;
```

Comment:
can we get this from consts + platform?

It looks like we specify decimals by chain here, maybe it should be platform?
https://github.com/wormhole-foundation/connect-sdk/blob/develop/core/base/src/constants/decimals.ts#L18